media-source-respec.html

```diff
@@ -74,15 +74,24 @@
     </section>
     <section id="sotd">
       <p>
-        On top of editorial updates, substantives changes since publication as a W3C Recommendation
-        in <a href="https://www.w3.org/TR/2016/REC-media-source-20161117/">November 2016</a> are
-        the addition of a {{SourceBuffer/changeType()}} method to switch codecs, the possibility to
-        create and use {{MediaSource}} objects off the main thread in dedicated workers, the
-        removal of the {{URL/createObjectURL()}} extension to the {{URL}} object following its
-        integration in the File API [[FILEAPI]], and the addition of {{ManagedMediaSource}},
-        {{ManagedSourceBuffer}}, and {{BufferedChangeEvent}} interfaces supporting power-efficient
-        streaming and active buffered media cleanup by the user agent. For a full list of changes
-        done since the previous version, see the <a href=
+        On top of editorial updates, substantive changes since publication as a W3C Recommendation
+        in <a href="https://www.w3.org/TR/2016/REC-media-source-20161117/">November 2016</a> are:
+        <ul>
+          <li>the addition of a {{SourceBuffer/changeType()}} method to switch among codecs or
+          bytestreams
+          </li>
+          <li>the possibility to create and use {{MediaSource}} objects off the main thread in
+          dedicated workers
+          </li>
+          <li>the removal of the {{URL/createObjectURL()}} extension to the {{URL}} object following
+          its integration in the File API [[FILEAPI]]
+          </li>
+          <li>the addition of {{ManagedMediaSource}}, {{ManagedSourceBuffer}}, and
+          {{BufferedChangeEvent}} interfaces supporting power-efficient streaming and active buffered
+          media cleanup by the user agent
+          </li>
+        </ul>
+        For a full list of changes made since the previous version, see the <a href=
         "https://github.com/w3c/media-source/commits/main">commits</a>.
       </p>
```

Comment:
```suggestion
        On top of editorial updates, substantive changes since publication as a W3C Recommendation
        in <a href="https://www.w3.org/TR/2016/REC-media-source-20161117/">November 2016</a> are:
        </p>
        <ul>
          <li>the addition of a {{SourceBuffer/changeType()}} method to switch among codecs or
          bytestreams
          </li>
          <li>the possibility to create and use {{MediaSource}} objects off the main thread in
          dedicated workers
          </li>
          <li>the removal of the {{URL/createObjectURL()}} extension to the {{URL}} object following
          its integration in the File API [[FILEAPI]]
          </li>
          <li>the addition of {{ManagedMediaSource}}, {{ManagedSourceBuffer}}, and
          {{BufferedChangeEvent}} interfaces supporting power-efficient streaming and active buffered
          media cleanup by the user agent
          </li>
        </ul>
        <p>
        For a full list of changes made since the previous version, see the <a href=
        "https://github.com/w3c/media-source/commits/main">commits</a>.
      </p>
```